### ISSUE TITLE
[bitnami/postgresql-ha] Question: How do I specify pgpool.serviceLabels?

### ISSUE THREAD

**Issue by jmaaks** [bitnami/charts#7306]
I see that pgpool.serviceLabels is available to add custom labels to the pgpool service.  What is the proper syntax to do this on the Helm command line?  I've tried variations of the below and I can't get it to work.  

--set pgpool.serviceLabels={label_name: value}

This gives me the following error:

Error: Service "postgresql-ha-pgpool" is invalid: metadata.labels: Invalid value: "pgpool {}": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')

I'm sure it's something simple where I have the variable value wrong!

**Comment by alvneiayu** [comment]
hi @jmaaks 

The way to set up serviceLabels using Helm is like this (example):

```
--set "pgpool.serviceLabels.app\.kubernetes\.io\/hello=world.test","pgpool.serviceLabels.app\.kubernetes\.io\/test-internal=True"
```

Testing the command, I found a bug in our chart. My colleague @rafariossaa  has included a change and right now is merged. Could you test with the last postgresql-ha version (7.9.2), please?

Thanks a lot and let me know if you still have problems.

Álvaro

**Comment by jmaaks** [comment]
Hi @alvneiayu ,

Has 7.9.2 been published?  We've tried adding "--version 7.9.2" when running helm and it can't find that version.
